### PR TITLE
Add Chile (SING)

### PR DESCRIPTION
**Pull request by systemcatch**
This is the parser for Chile's northern grid area (SING) which accounts for about 19% of the total electricity production.  Production breakdown is available for solar and wind with every other source being combined together.  However the region has little hydro and no nuclear or geothermal so it shouldn't be an issue to label it as 'unknown'.

ref #655 @jarek has done the groundwork for generating the region on the map.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +126,9 @@ Production capacities are centralized in the [zones.json](https://github.com/tmr
 - Canada (Ontario): [Gridwatch](http://live.gridwatch.ca/total-capacity.html)
 - Canada (Québec): [Hydro-Québec](http://www.hydroquebec.com/generation/)
 - Canada (Saskatchewan): [SaskPower](http://www.saskpower.com/our-power-future/our-electricity/)
+- Chile (SING):
+[CIGRE](http://www.cigre.org/var/cigre/storage/original/application/6a789c1375a22a39e14a284bee7ff988.pdf)
```
Comment (corradio) [inline README.md]: I think this lacks the `- ` prefix

**Comment by systemcatch** [comment]
Based on studying ne_10m_admin_1_states_provinces_lakes on mapshaper we can't split Chile into it's 4 grid regions (SING, SIC, SEA, SEM) using the existing countrytopos.js.  The reason is that 3 states in the SIC region have overlapping code_hasc and fips values that cannot be split uniquely.  We'll have to create a new function in countrytopos.js to split by some other identifier.  Possibly adm1_code or gn_a1_code.

**Comment by jarek** [comment]
For what it's worth, Philippines will be easiest to split based on region_cod in the Natural Earth files.

So it's looking like we'd best extenf topogen.sh to take in a choice of identifiers...

**Comment by corradio** [comment]
Remember to run `PYTHONPATH=. python mockserver/update_state.py CL-SING` in order to preview in the frontend!
image

**Pull request merged**